css/site.styl

```diff
@@ -154,31 +148,90 @@ $speed-transition-fast = .1s
 
 html
   font-family: "Montserrat", font-stack('Helvetica')
-  font-size: $font-size-basic
-  letter-spacing: $font-tracking-basic
-  text-transform: uppercase
+  font-size: 12px
 
 body
   line-height: $font-leading-basic
   
 h1,h2,h3,h4,h5
   margin: 0
+  
+.font-nav 
+  font-size: 11px   
```

Comment:
when Clay is specing Font Size I'm wondering if be means in pt rather than pixels. Maybe doesn't matter as everything is now relative and you have bumped the sizes on larger screens?